package.json

```diff
@@ -6,27 +6,30 @@
   "files": [
     "dist"
   ],
-  "main": "./dist/prime.umd.js",
-  "module": "./dist/prime.es.js",
+  "main": "./dist/prime.umd.cjs",
+  "module": "./dist/prime.js",
+  "types": "./dist/main.d.ts",
   "exports": {
     ".": {
-      "import": "./dist/prime.es.js",
-      "require": "./dist/prime.umd.js"
+      "import": "./dist/prime.js",
+      "require": "./dist/prime.umd.cjs"
+    },
+    "./theme": {
+      "import": "./dist/theme.js"
     },
     "./prime.css": "./dist/prime.css"
   },
   "homepage": "http://viamrobotics.github.io/prime",
   "repository": "git://github.com/viamrobotics/prime.git",
   "scripts": {
     "start": "vite dev",
-    "build": "vite build --mode production",
-    "postbuild": "node ./scripts/postbuild",
+    "build": "npm run build-bundles && npm run build-types && npm run build-storybook",
+    "build-bundles": "vite build --mode production",
+    "build-types": "tsc --project tsconfig.build.json",
```

Comment:
It is possible to extend `HTMLElement` types or `JSXIntrinsicElement` types for much better type safety when adding attributes to custom elements, but probably not worth it anymore since we want to move this to a purely Svelte component library